website/pages/en/cookbook/sorting.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: GraphQL Sorting Strategies
+---
+
+## Sorting Entity IDs
+
+When creating subgraphs in The Graph, it's important to consider the type of ID used for your entities. This choice can have a significant impact on the performance and ease of use of your subgraph.
```

Comment:
This is actually no a big consideration. As of now, IDs should always be `Bytes`.

Reply:
Unless there is a good reason to use `ID` which translates to string. Examples might be traits or unique names.